.travis.yml

```diff
@@ -24,6 +24,6 @@ script:
     - export PATH="/tmp/dummy_bin:$PATH"
     # Setup Avocado-vt for functional tests
     - make install
-    - AVOCADO_LOG_DEBUG=yes avocado vt-bootstrap --yes-to-all
+    - AVOCADO_LOG_DEBUG=yes avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
```

Comment:
... and if you want to be super-clear I'd support:

```
--no-download             deprecated, symlink to --no-download-assets
--no-download-assets      skip downloading/extracting asset files (JeOS)
--no-download-providers   skip checking/downloading providers (test repos)
```

To me this would be easier to understand and it allows the same functionality.